source/localizable/tutorial/acceptance-test.md

```diff
@@ -0,0 +1,140 @@
+
+```/tests/acceptance/list-rentals-test.js
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | homepage');
+
+test('should list available rentals.', function (assert) {
+});
+
+test('should link to information about the company.', function (assert) {
+});
+
+test('should link to contact information.', function (assert) {
+});
+
+test('should filter the list of rentals by city.', function (assert) {
+});
+```
+
+These tests will fail, since Ember tests will always fail without assertions.
+Since we have an idea of what we want our application to look like, we can also add some details to the tests.
+
+Ember acceptance tests provide test helpers to perform common tasks, such as visiting routes, filling in fields, clicking on elements, and waiting for pages to render.
+
+To check that rentals are listed, we'll first visit the index route and check that the results show 3 listings.
+
+```/tests/acceptance/list-rentals-test.js
+test('should list available rentals.', function (assert) {
+  visit('/');
+  andThen(function () {
+    assert.equal(this.$('.results .listing').length, 3, "should see 3 listings");
+  });
+});
+```
+The test assumes that we show the list of rentals in an element with class `results`, and that each rental will have an element with class `listing`.
+
+The [`visit`](http://emberjs.com/api/classes/Ember.Test.html#method_visit) helper loads the route specified for the given URL.
+
+The [`andThen`](../../testing/acceptance/#toc_wait-helpers) helper waits for all previously called test helpers to complete before executing the function you provide it.
+In this case we need to wait for the page to load after `visit`, so that we can assert that the listings are displayed.
+
+For the next two tests, we want to verify that clicking the about and contact page links successfully load the proper URLs.
+We'll use the [`click`](http://emberjs.com/api/classes/Ember.Test.html#method_click) helper to simulate a user clicking these links.
+After the new screen loads, we just verify that the new URL matches our expectation using the [`currentUrl`](http://emberjs.com/api/classes/Ember.Test.html#method_currentURL) helper.
+
+```/tests/acceptance/list-rentals-test.js
+test('should link to about page', function (assert) {
+  visit('/');
+  click('a:contains("About")');
+  andThen(function () {
+    assert.equal(currentURL(), '/about', "should navigate to about");
+  });
+});
+
+test('should link to contacts page', function (assert) {
+  visit('/');
+  click('a:contains("Contact")');
+  andThen(function () {
+    assert.equal(currentURL(), '/contact', "should navigate to contact");
+  });
+});
+```
+Note that we can call 2 [asynchronous test helpers](../../testing/acceptance/#toc_asynchronous-helpers) in a row without needing to use `andThen` or a promise.
```

Comment:
we can call *two*

Using a number instead of a word for two looks out of place here (I think there's a grammatical rule for this, but don't remember what it's called 😁 )